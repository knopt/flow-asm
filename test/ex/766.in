6
5
-7.268685624298157
3.691852335490582 -5.027790009026154 -1.81503129667915 6.128083222014318 -8.962990012617222 -4.284356672096323 
-5.230521464356523 5.336295612609796 -6.233117711054046 9.842598648662559 -7.372017581527681 6.93614425981114 
6.308267412037001 -2.744371477028354 -2.4796328338824676 9.727466488118644 -0.21272897503544996 5.43347962527471 
4.126113034534189 8.106439228938935 -7.343885046254286 -5.971165211867544 2.0441428629202445 -0.22857561204472887 
5.823947494642429 -8.144193499081984 -4.115528855505291 -7.658755059661615 4.758842670107182 5.24003548402254 

-6.749060512283929 -9.701207231748281 9.129970010625264 -4.203476648478508 -8.216900797796871 
