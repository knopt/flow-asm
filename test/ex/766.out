241.7874351719803 -142.26781791966397 1.6704358769648402 126.71412417453868 -266.9114474856382 -29.39189912457877 
-214.80438609182897 221.10025963336608 -183.8438765315097 328.83050544952323 -395.39275351465295 370.94708603130357 
278.3058224138594 -238.0204674599936 -71.67406572788224 451.8273493320926 -68.06351120388047 194.4224544499334 
89.81488666879089 263.6911117967794 -206.08528897225867 -136.70204328798735 71.65674052469164 -134.02072587790911 
193.1096782813175 -316.98396034087887 -40.20416955784057 -47.96860363979793 192.74381027989534 71.71722838371304 
